packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -382,50 +392,66 @@ describe('FLE2', function () {
+        ['unindexed', collectionNameUnindexed],
+      ]) {
+        it(`can edit and query the ${mode} encrypted field in the CRUD view`, async function () {
+          await browser.shellEval(`db.createCollection('${coll}')`);
+          await browser.shellEval(
+            `db[${JSON.stringify(
+              coll
+            )}].insertOne({ "phoneNumber": "30303030", "name": "Person X" })`
+          );
 
-        await browser.navigateToCollectionTab(
-          databaseName,
-          collectionName,
-          'Documents'
-        );
+          await browser.navigateToCollectionTab(
+            databaseName,
+            coll,
+            'Documents'
+          );
 
-        const result = await getFirstListDocument(browser);
-        expect(result.phoneNumber).to.be.equal('"30303030"');
+          const result = await getFirstListDocument(browser);
+          expect(result.phoneNumber).to.be.equal('"30303030"');
 
-        const document = await browser.$(Selectors.DocumentListEntry);
-        const value = await document.$(
-          `${Selectors.HadronDocumentElement}[data-field="phoneNumber"] ${Selectors.HadronDocumentClickableValue}`
-        );
-        await value.doubleClick();
+          const document = await browser.$(Selectors.DocumentListEntry);
+          const value = await document.$(
+            `${Selectors.HadronDocumentElement}[data-field="phoneNumber"] ${Selectors.HadronDocumentClickableValue}`
+          );
+          await value.doubleClick();
 
-        const input = await document.$(
-          `${Selectors.HadronDocumentElement}[data-field="phoneNumber"] ${Selectors.HadronDocumentValueEditor}`
-        );
-        await input.setValue('10101010');
+          const input = await document.$(
+            `${Selectors.HadronDocumentElement}[data-field="phoneNumber"] ${Selectors.HadronDocumentValueEditor}`
+          );
+          await input.setValue('10101010');
 
-        const footer = await document.$(Selectors.DocumentFooterMessage);
-        expect(await footer.getText()).to.equal('Document modified.');
+          const footer = await document.$(Selectors.DocumentFooterMessage);
+          expect(await footer.getText()).to.equal('Document modified.');
 
-        const button = await document.$(Selectors.UpdateDocumentButton);
-        await button.click();
-        await footer.waitForDisplayed({ reverse: true });
+          const button = await document.$(Selectors.UpdateDocumentButton);
+          await button.click();
+          try {
+            await footer.waitForDisplayed({ reverse: true, timeout: 1000 });
+          } catch (err) {
+            if (
+              mode === 'unindexed' &&
+              (await footer.getText()) ===
+                'Found indexed encrypted fields but could not find __safeContent__'
+            ) {
+              return; // This version is affected by SERVER-68065 where updates on unindexed fields in QE are buggy.
+            }
+          }
```

Comment:
So if the field is encrypted (listed in `fleEncryptedFieldsMap`), but does not have `queries: { queryType: 'equality' }` property specified in `fleEncryptedFieldsMap` the filed should be editable but not searchable?

Reply:
> should be editable but not searchable

Yes :+1:  a field without `queryType: 'equality'` (or, in the future, `queryType: 'range'`) is not searchable by definition, but the feedback from the QE product team was that it would be better for these fields to be editable (and accept the caveat that updates to these fields won’t be able to verify that the field hasn’t been updated in the background)